lib/Github/Api/User.php

```diff
@@ -145,6 +145,8 @@ public function subscriptions($username)
     }
 
     /**
+     * @deprecated use repos($username, $options) instead
```

Comment:
You should remove this function. It is okey, we will soon release a new major version. 

Please make sure to add this in the changelog.